src/test/kotlin/EquivalentUnits.kt

```diff
@@ -43,6 +43,7 @@ class EquivalentUnits {
             "pressure:pa",
             "pressure:megapa",
             "pressure:n-per-millim2",
+            "standard_volume:scm_1atm_15deg_c_iso",
```

Comment:
The ones that needs to be added here is `volume:kilo-m3` as it is equivalent to `volume:megal` (being equivalent, means they share the same conversion factors for their quantity)
```suggestion
            "volume:kilo-m3",
            "volume:megal",
```